Formula/bash-snippets.rb

```diff
@@ -1,20 +1,58 @@
 class BashSnippets < Formula
   desc "Collection of small bash scripts for heavy terminal users"
   homepage "https://github.com/alexanderepstein/Bash-Snippets"
-  url "https://github.com/alexanderepstein/Bash-Snippets/archive/v1.16.0.tar.gz"
-  sha256 "19a029a204f16bd4db30cf7fa32aa6953dd10dcb996138b9bc65c01612afaac7"
+  url "https://github.com/alexanderepstein/Bash-Snippets/archive/v1.16.1.tar.gz"
+  sha256 "19314a4dc165d4cd13e56acbd8abaf4d447ba8f1ac58f8b7b54c985458135e77"
+
+  conflicts_with "cheat", :because => "Both install a `cheat` executable"
 
   bottle :unneeded
 
+  option "with-cheat", "Install cheat"
+  option "with-cloudup", "Install cloudup"
+  option "with-crypt", "Install crypt"
+  option "with-cryptocurrency", "Install cryptocurrency"
+  option "with-currency", "Install currency"
+  option "with-geo", "Install geo"
+  option "with-movies", "Install movies"
+  option "with-newton", "Install newton"
+  option "with-qrify", "Install qrify"
+  option "with-short", "Install short"
+  option "with-siteciphers", "Install siteciphers"
+  option "with-stocks", "Install stocks"
+  option "with-taste", "Install taste"
+  option "with-todo", "Install todo"
+  option "with-weather", "Install weather"
+  option "with-ytview", "Install ytview"
+
   def install
-    system "./install.sh", "--prefix=#{prefix}", "all"
+    alltools = %w[cheat cloudup crypt cryptocurrency currency geo movies newton qrify short siteciphers stocks taste todo weather ytview]
+    snippets = %w[]
+
+    alltools.each do |tool|
+      snippets.push(tool.to_s) if build.with?(tool.to_s)
+    end
+
+    if snippets.empty?
+      alltools.each do |snippet|
```

Comment:
I think we just want `system "./install.sh", "--prefix=#{prefix}", "all"` if it's empty